openwisp_controller/config/static/config/js/widget.js

```diff
@@ -341,6 +348,26 @@
         });
         // so that other files can use updateContext
         window.updateContext = updateContext;
+
+        $('.jsoneditor').on('input paste', '.has-max-length:visible', function(e){
+            var field = $(e.target),
+            value;
+
+            if (e.originalEvent.type === 'paste') {
+                value = e.originalEvent.clipboardData.getData('text');
+                if (field.val().indexOf('{{') > -1){
```

Comment:
I added this because with previous code when I try to paste the config variable (say `OW_subnet_ip_add_1`) after adding `{{` in the field then the variable value is equal to `openwisp_subnet_ip_add1` which do not contains `{{` due to which max length attr is removed but it is actually present in the field. So I added this conditional check to avoid this condition.